src/lib.rs

```diff
@@ -209,6 +209,11 @@ pub use types::string::Str;
 pub use types::timestamp::Timestamp;
 pub use types::IonType;
 
+pub use ion_data::IonData;
+pub mod ion_eq {
+    pub use crate::ion_data::IonEq;
+}
```

Comment:
Why do we want a top-level trivial module for `ion_eq`?  Is this just to maintain backwards compatibility?  I would get rid of it if we can.

Reply:
This was just for backwards compatibility. I can get rid of it.